Wrong hyperlink in How to disable BGP Propagation

surajmb opened the pull request:
Here is the right link. 

https://docs.microsoft.com/en-us/azure/virtual-network/manage-route-table#create-a-route

azurecla commented:
Hi __@timothywarner__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@timothywarner : Thanks for your contribution to the Azure documentation! The author, @GarethBradshawMSFT, has been notified to review your proposed change.

timothywarner commented:
Hello? :) -Tim

telmosampaio commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

Ja-Dunn commented:
@telmosampaio @GarethBradshawMSFT - The changes look good for the most part, but the markdown for the notes is actually correct in the original version. Making these changes would result in incorrect formatting of the notes. Please revert. Thanks!

jimdial commented:
@timothywarner Thank you for the edits. The change you made for the note formatting is incorrect, so I'm going to close this PR.  Separately, we will incorporate the on-premise to on-premises change you submitted. @GarethBradshawMSFT 

Thanks

#please-close

surajmb committed:
Wrong hyperlink in How to disable BGP Propagation

Here is the right link. 

https://docs.microsoft.com/en-us/azure/virtual-network/manage-route-table#create-a-route

ktoliver committed:
fix relative link formatting
